sv.c

```diff
@@ -3105,6 +3106,11 @@ Perl_sv_2pv_flags(pTHX_ SV *const sv, STRLEN *const lp, const U32 flags)
 	    if (lp)
 		*lp = len;
 	    SAVEFREEPV(buffer);
+
+            if (!FEATURE_STRINGIFICATION_IS_ENABLED) {
+                Perl_croak(aTHX_ "Attempt to stringify reference under no feature stringification");
+            }
```

Comment:
Wouldn't it be better to have this check _before_ doing all the work to stringify the reference?

Reply:
I originally planned to add more information to the message, probably using the information used to produce the typical `Class=TYPE(0xaddress)`, but later decided to keep it simple, depending on any feedback.